src/main/java/org/jenkinsci/test/acceptance/junit/CspRule.java

```diff
@@ -41,7 +41,7 @@ public void evaluate() throws Throwable {
                             && !isSkipped()
                             && !d.getTestClass().getName().equals("plugins.ArtifactoryPluginTest")) {
                         ContentSecurityPolicyReport csp = new ContentSecurityPolicyReport(jenkins);
-                        csp.open();
+                        jenkins.runThenHandleUserPrompt(() -> csp.open());
```

Comment:
Avoids the need for copypasta by dismissing the user prompt inside of `CspRule`. There are likely other ways of doing this; perhaps as of https://github.com/mozilla/geckodriver/issues/1151 `driver.quit()` could be used instead, although that means that the driver has to be reopened and it wasn't immediately obvious to me how to implement that. Since this existing PR is working, I didn't see a need to spend more time thinking about a potential alternative implementation with no clear benefit.